src/main/java/org/jitsi/jicofo/bridge/BridgeSelectionStrategy.java

```diff
@@ -86,61 +93,216 @@ public Bridge select(
     }
 
     /**
-     * Selects a bridge to be used for a new participant in a conference,
-     * assuming that no other bridge is used by the conference (i.e. this
-     * is the initial selection of a bridge for the conference).
+     * A1. (Happy case 1): There is a NOL bridge in the conference and in the
+     * region. Use the least loaded of them.
      *
-     * @param bridges the list of bridges to select from.
-     * @param participantRegion the region of the participant for which a
-     * bridge is to be selected.
-     * @return the selected bridge, or {@code null} if no bridge is
-     * available.
+     * @param bridges
+     * @param conferenceBridges
+     * @param participantRegion
+     *
+     * @return
      */
-    private Bridge selectInitial(List<Bridge> bridges,
-                                 String participantRegion)
+    private Optional<Bridge> findNotLoadedBridgeAlreadyInConferenceInRegion(List<Bridge> bridges,
+                                                                            List<Bridge> conferenceBridges,
+                                                                            String participantRegion)
     {
-        Bridge bridge = null;
+        Optional<Bridge> result = bridges.stream()
+            .filter(not(Bridge::isOverloaded))
+            .filter(selectFrom(conferenceBridges))
+            .filter(inRegion(participantRegion))
+            .findFirst();
 
-        // Prefer a bridge in the participant's region.
-        if (participantRegion != null)
+        if (result.isPresent())
         {
-            bridge = findFirstOperationalInRegion(bridges, participantRegion);
+            logger.info("bridge_selected,rule=a1,bridge=" + result.get() +
```

Comment:
I don't think we need to refer to `rule=a1` here (the method name will now give enough context)